app/scripts/store/dimStores.directive.js

```diff
@@ -22,7 +22,7 @@
         "    'hide-filtered': vm.hideFilteredItems,",
         "    'show-elements': vm.showElements",
         '  }">',
-        '  <div dim-store-heading store-data="store"></div>',
+        '  <div dim-store-heading class="character dim-col-{{vm.charCol}}" store-data="store"></div>',
```

Comment:
If we remove `dim-col-{{vm.charCol}}` here, the vault header spans the width of the vault. not sure which I prefer.

![screen shot 2016-05-11 at 1 00 07 pm](https://cloud.githubusercontent.com/assets/424158/15189394/6f3e5774-1778-11e6-87eb-cccf9ee7961d.png)

vs

![screen shot 2016-05-11 at 1 01 29 pm](https://cloud.githubusercontent.com/assets/424158/15189402/81fd504a-1778-11e6-90a2-4c7bf5a684a8.png)